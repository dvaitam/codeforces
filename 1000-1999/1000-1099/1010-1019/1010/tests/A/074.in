7
13
5 2 3 2 8 9 6
8 9 8 6 10 4 2
