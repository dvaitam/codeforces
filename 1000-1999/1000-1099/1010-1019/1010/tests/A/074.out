772.9125364431
